src/style/foundation/video-calling.less

```diff
@@ -272,16 +276,16 @@
     animation:
       flyUp 4s linear forwards,
       fadeIn 1s;
-    font-size: 2rem;
+    font-size: 1.75rem;
     gap: 0.1rem;
   }
 
   .emoji-text {
```

Comment:
Correct me if I'm wrong, but as I'm looking [at design](https://www.figma.com/design/jnygr7PEOk1yHgjBF18udc/Desktop?node-id=3312-61738&node-type=frame&t=7NqH7vrGaNIYxOqC-0) I see the button should be blue, instead of green, as you've posted in PR's description. 
 
<img width="401" alt="Screenshot 2024-10-23 at 13 19 32" src="https://github.com/user-attachments/assets/706faeca-38f7-40c3-9fe8-2a06f9f78fc0">

Reply:
it should use the accent color of the account @olafsulich

Reply:
got it!